api/app/schemas/project_schemas.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional
+from uuid import UUID
+
+from pydantic import BaseModel, HttpUrl
+
+from app.schemas.base_schema import BaseSchema
+
+
+class ProjectCreate(BaseSchema):
+    name: str
+    avatar_url: Optional[HttpUrl] = None
+    description: Optional[str] = None
+    organization_id: str
```

Comment:
since we want to set org (not let the user set it) this should be excluded from the `ProjectCreate` schema - so ideally the flow is 
1. user specifies the project name/avatar/description
2. we verify that user _can_ create projects (skip this for now)
3. we get the requesting user's org, apply that when creating in the Controller
4. we return the created Project with the user's org applied

Otherwise I could pick any org ID (not my own) and create projects for that instead

Reply:
should this be in an actual controller or can the model handle this?

is it ever going to be the case when you **don't** attach the user's org (or current org?) to a project? 
or, in other words, can a project ever be created where it won't be attached to the user's current org?

I guess if they wan't to created a project for a different org, while actively working on a totally different org (e.g. working on Org A but wanna create a project for Orb B) — but maybe that's too convoluted?

Reply:
I think just the logic to put together the model call from the 2 request params would live in the controller. You'll always get 2 params (because the user is coming from the auth header, the project is the post body) so we'll need to stitch them together somewhere. Technically you could just do it in the router, but if we're following the "data formatting only in the router, business logic in the controller" pattern, then it goes to the controller. But yeah the controller should just build the call to create the model with that user as the actor